beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -328,3 +326,15 @@ func (s *Service) hasBlockAndState(ctx context.Context, blockRoot [32]byte) bool
 	hasState := hasStateSummary || s.cfg.beaconDB.HasState(ctx, blockRoot)
 	return hasState && s.cfg.chain.HasBlock(ctx, blockRoot)
 }
+
+func seenAttCacheKey(slot primitives.Slot, committeeID primitives.CommitteeIndex, aggregationBits []byte) string {
```

Comment:
A benchmark test would be nice so that we can see the before/after improvements.

Reply:
Benchmark between old and new implementations:

```
goos: linux
goarch: amd64
cpu: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
BenchmarkCacheKeyImplementations
BenchmarkCacheKeyImplementations/Old_Cache_Key_Implementation
BenchmarkCacheKeyImplementations/Old_Cache_Key_Implementation-8                  7620234               169.0 ns/op           280 B/op          4 allocs/op
BenchmarkCacheKeyImplementations/New_Cache_Key_Implementation
BenchmarkCacheKeyImplementations/New_Cache_Key_Implementation-8                 35468606                31.01 ns/op           24 B/op          1 allocs/op
```